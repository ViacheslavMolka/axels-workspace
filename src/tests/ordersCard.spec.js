import React from 'react';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import OrdersCard from '../components/ordersCard';


configure({ adapter: new Adapter() });

it('OrdersCard snapshot', () => {
    const data = [{
        image_url: '',
        title: '',
        price: 30,
        color: ''
    }];
    const component = shallow(<OrdersCard data={data}/>);
    expect(component).toMatchSnapshot();
});