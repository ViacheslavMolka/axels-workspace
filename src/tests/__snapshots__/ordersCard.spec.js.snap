// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrdersCard snapshot 1`] = `ShallowWrapper {}`;
